pkgs/applications/misc/calibre/default.nix

```diff
@@ -129,14 +129,14 @@ stdenv.mkDerivation rec {
     ];
     categories = "Office";
     extraEntries = ''
-      Actions=ebook-edit ebook-viewer
+      Actions=Edit;Viewer;
```

Comment:
Kind of: https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#example.